Documentation/conf.py

```diff
@@ -77,9 +77,12 @@
 if branch is None or branch == 'latest':
     branch = 'HEAD'
     archive_name = 'master'
+    chart_release = './cilium'
 elif branch == 'stable':
     branch = release
     archive_name = release
+    chart_release = 'cilium/cilium --version ' + release
+    tags.add('stable')
```

Comment:
What does the `tags.add()` do? I don't see any tags variable declaration anywhere.

Reply:
Ah nice, this makes a variable accessible from sphinx is that right? That's really handy to know.

Reply:
Yep tags are necessary for the [only](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-only) directive to work so we could conditionally switch between templates for installation from the helm repo or from a local git clone.